src/main/resources/templates/settingsForSingleProject.vm

```diff
@@ -195,7 +195,7 @@ $webResourceManager.requireResource("de.uhd.ifi.se.decision.management.jira:styl
             });
 
             getProjectIssueTypes("$projectKey", function (issueTypes) {
-                html = "<button class=\"aui-button aui-dropdown2-trigger\" aria-controls=\"webhook-rootType\">Webhook Type</button>" +
+                html = "<button id=\"dropDownButto\" class=\"aui-button aui-dropdown2-trigger\" aria-controls=\"webhook-rootType\">Webhook Type</button>" +
```

Comment:
Thanks. I replaced the drop down menu with a select in the next commit because I think its easier.